### ISSUE TITLE
auth-server image not build

### ISSUE THREAD

**Issue by 0xffea** [uselagoon/lagoon#178]
```
$ make lagoon-kickstart
docker build --quiet --network host --build-arg IMAGE_REPO=lagoon -t lagoon/rabbitmq -f services/rabbitmq/Dockerfile services/rabbitmq
sha256:d59eb36141a9595c97f28d007db63fb10a9cde87da3eca2694d26b017d9eb28e
touch build/rabbitmq
docker build --quiet --network host --build-arg IMAGE_REPO=lagoon -t lagoon/commons -f images/commons/Dockerfile images/commons
sha256:1697510961c7830f781e8f08652b07afa577cfd377c4843937c7323c26c8275d
touch build/commons
docker build --quiet --network host --build-arg IMAGE_REPO=lagoon --build-arg NODE_VERSION=8 -t lagoon/node:8 -f images/node/Dockerfile images/node
sha256:a83665f19f7a40abdbfdd4b2c10c452c873b188c49a48700d56baf7085dd217c
touch build/node__8
docker build --quiet --network host --build-arg IMAGE_REPO=lagoon --build-arg NODE_VERSION=8 -t lagoon/node:8-builder -f images/node/builder/Dockerfile images/node/builder
sha256:8e79fbd5719157b43f5f257557907f18c2a64f7fcf6a356dfe0f93fd4055b3c2
touch build/node__8-builder
docker build --quiet --network host --build-arg IMAGE_REPO=lagoon -t lagoon/yarn-workspace-builder -f images/yarn-workspace-builder/Dockerfile .
sha256:baa3d659d17e200331c944e19a039be35bad7526083fce349cd857145cc9d810
touch build/yarn-workspace-builder
docker build --quiet --network host --build-arg IMAGE_REPO=lagoon -t lagoon/openshiftremove -f services/openshiftremove/Dockerfile services/openshiftremove
sha256:926fc643a15b2bcf9e126288e4c646b0c8ce4b832c3ea7435c9f0615d3419b38
touch build/openshiftremove
docker build --quiet --network host --build-arg IMAGE_REPO=lagoon -t lagoon/openshiftbuilddeploy -f services/openshiftbuilddeploy/Dockerfile services/openshiftbuilddeploy
sha256:080c0eabc9aa0a930036d0167600c3a90b872e582fdf8ab7249ab6a91c80c7e2
touch build/openshiftbuilddeploy
docker build --quiet --network host --build-arg IMAGE_REPO=lagoon -t lagoon/openshiftbuilddeploymonitor -f services/openshiftbuilddeploymonitor/Dockerfile services/openshiftbuilddeploymonitor
sha256:fefb7020d8f753b2aa4c0c0a7145692f7a8decb2119f6d7adbf7a40269b1cf68
touch build/openshiftbuilddeploymonitor
docker build --quiet --network host --build-arg IMAGE_REPO=lagoon -t lagoon/logs2slack -f services/logs2slack/Dockerfile services/logs2slack
sha256:656eb21c82e7751b491f314e7028e0ce92263d214bd365bfb1f0160dc82d04e1
touch build/logs2slack
docker build --quiet --network host --build-arg IMAGE_REPO=lagoon -t lagoon/api -f services/api/Dockerfile services/api
sha256:ce0482cbb155982c0463cb391ef546e871a55d75d50b3c89ea292a65014adb7e
touch build/api
docker build --quiet --network host --build-arg IMAGE_REPO=lagoon -t lagoon/ssh -f services/ssh/Dockerfile .
sha256:f26f25a23d2cd8261933f182c1b9f6cbab911130d1e7c67a15bd169911371e68
touch build/ssh
docker build --quiet --network host --build-arg IMAGE_REPO=lagoon -t lagoon/local-git -f local-dev/git/Dockerfile local-dev/git
sha256:2a31bd3ee2a213b8a63be86b04ddd8cf43e42863f8ab45b42dea5df018a538d4
touch build/local-git
docker build --quiet --network host --build-arg IMAGE_REPO=lagoon -t lagoon/local-api-data-watcher-pusher -f local-dev/api-data-watcher-pusher/Dockerfile local-dev/api-data-watcher-pusher
sha256:b276049c9b60f490216f74340b25892bf0f7218ab6fe0a96a22231c7aa10e019
touch build/local-api-data-watcher-pusher
docker build --quiet --network host --build-arg IMAGE_REPO=lagoon -t lagoon/rest2tasks -f services/rest2tasks/Dockerfile services/rest2tasks
sha256:86d4a43cfea009ddec3343ecd558b677b6274d0aac23c91c8c1dfa4c9fed60b0
touch build/rest2tasks
IMAGE_REPO=lagoon CI_USE_OPENSHIFT_REGISTRY=false docker-compose -p lagoon up -d rabbitmq openshiftremove openshiftbuilddeploy openshiftbuilddeploymonitor logs2slack api ssh local-git local-api-data-watcher-pusher rest2tasks
Pulling auth-server (lagoon/auth-server:latest)...
ERROR: repository lagoon/auth-server not found: does not exist or no pull access
Makefile:423: recipe for target 'lagoon-kickstart' failed
make: *** [lagoon-kickstart] Error 1
```

**Comment by rtprio** [comment]
Were you able to continue kickstarting after a `make build/auth-server` ?

**Comment by 0xffea** [comment]
yes.

**Comment by Schnitzel** [comment]
added a pull request https://github.com/amazeeio/lagoon/pull/181

**Comment by Schnitzel** [comment]
fixed in #181
